source/fuzz/fuzzer_util.cpp

```diff
@@ -1616,6 +1616,45 @@ bool IdUseCanBeReplaced(opt::IRContext* ir_context,
     return false;
   }
 
+  if (ir_context->get_feature_mgr()->HasCapability(SpvCapabilityShader)) {
```

Comment:
Add a comment here, something like:

```
// With the Shader capability, memory scope and memory semantics operands are required to be constants, so they cannot be replaced arbitrarily.
```

Reply:
Done.

Reply:
Please move the comment so that it is inside the if statement - to make it clear that it relates to the code in the body of the if statement.

Reply:
Done.